src/sharepoint/account/add-legacy/sharepoint-account-add-legacy.html

```diff
@@ -0,0 +1,51 @@
+<div data-ng-controller="SharepointAccountAddLegacyCtrl as $ctrl">
+    <div data-wizard
+         data-wizard-title="'sharepoint_accounts_action_sharepoint_add_title' | translate"
+         data-wizard-on-cancel="resetAction"
+         data-wizard-on-finish="submit">
+
+        <div data-wizard-step
+             data-wizard-step-valid="$ctrl.addAccountForm.$valid">
+
+            <p data-translate="sharepoint_accounts_action_sharepoint_add_intro"></p>
+
+            <oui-spinner data-ng-if="$ctrl.loading"></oui-spinner>
+
+            <form class="form-horizontal" name="$ctrl.addAccountForm" novalidate
+                  data-ng-if="!$ctrl.loading">
+                <div class="form-group"
+                     data-ng-repeat="option in $ctrl.optionsList track by $index"
+                     data-ng-class="{
+                        'has-error': $ctrl.addAccountForm.$invalid && $ctrl.addAccountForm.$dirty,
+                        'has-success': $ctrl.addAccountForm.$valid && $ctrl.addAccountForm.$dirty}">
+                    <div class="clearfix">
```

Comment:
bootlint: E014  Columns (`.col-*-*`) can only be children of `.row`s or `.form-group`s 

Documentation: https://github.com/twbs/bootlint/wiki/E014

You can get rid of `<div class="clearfix">`.

Reply:
That's legacy, you know ;)